.github/workflows/docker_build_images.yml

```diff
@@ -8,14 +9,81 @@ on:
   workflow_dispatch:
 
 concurrency:
-  group: format('{0}-{1}', ${{ github.ref }}, 'Docker Build Images (v20+)')
+  group: format('{0}-{1}', ${{ github.ref }}, 'Build Docker Images')
   cancel-in-progress: true
 
 permissions: read-all
 
 jobs:
+  build_and_push_vttestserver:
+    name: Build and push vttestserver
+    runs-on: gh-hosted-runners-16cores-1
+    if: github.repository == 'vitessio/vitess'
+
+    strategy:
+      fail-fast: true
+      matrix:
+        branch: [ mysql80 ]
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+#      - name: Login to Docker Hub
```

Comment:
Need to fix the indentation here or do we want to remove this section?

Reply:
I am currently testing this, sorry this is still a WIP but did not mark it as a draft.